moped-editor/src/layouts/DashboardLayout/NavBar/NavigationSearchInput.js

```diff
@@ -198,8 +206,10 @@ const NavigationSearchInput = ({ input404Class }) => {
     switch (key) {
       // On Escape key, clear the search
       case "Escape":
-        setSearchTerm("");
         setSearchResultsAnchor(null);
+        // Slide away the search bar and reset the input
```

Comment:
I made this change because I noticed that the search box stops responding when:
- enter a search term to populate results
- press ESC while the results are showing
- the input remains open but no longer responds to searches